google-cloud-speech/README.md

```diff
@@ -19,6 +19,13 @@ steps:
 $ gem install google-cloud-speech
 ```
 
+### Migration Guide
+
+This release, 0.30.0, introduces breaking changes relative to the previous
```

Comment:
I would recommend future-proofing this sentence with: "The 0.30.0 release introduced ..."

Reply:
Isn't the intention to remove the warning in the 0.31.0 release? That's what we did for Natural Language.

Reply:
Maybe so, but future-proofed phrasing is a good way to go just in case it doesn't happen.

Reply:
Given the time-frame of changes for NL, what was the time delta between 0.30.0 and 0.31.0?